server/helpers/blacklistToken.js

```diff
@@ -0,0 +1,61 @@
+import cron from 'node-cron';
+import jwt from 'jsonwebtoken';
+import { Op } from 'sequelize';
+import models from '../db/models';
+
+const { TokenBlacklist } = models;
+/**
+  * @Module TokenBlacklist
+  * @description Authentication related methods
+  */
+class Blacklist {
+  /**
+   * @static
+   * @param {string} token - user token
+   * @returns {null} - no value
+   * @memberof Blacklist
+   */
+  static async addToBlacklist(token) {
+    const { exp } = jwt.decode(token);
+    await TokenBlacklist.create({
+      token,
+      expires: exp
+    });
+  }
+
+  /**
+   * @static
+   * @param {string} token - user token
+   * @returns {null} - null
+   * @memberof Blacklist
+   */
+  static async checkBlacklist(token) {
+    const blacklistedToken = await TokenBlacklist.findOne({
+      where: { token }
+    });
+    return blacklistedToken;
+  }
+
+  /* istanbul ignore next */
+  /**
+   * @static
+   * @returns {null} - null
+   * @memberof Blacklist
+   */
+  static async startCronJob() {
+    cron.schedule('59 23 * * *', () => {
```

Comment:
Can you add a comment explaining what this does.

Reply:
okay